packages/playwright/src/reporters/blob.ts

```diff
@@ -111,6 +111,11 @@ export class BlobReporter extends TeleReporterEmitter {
 
   private _defaultReportName(config: FullConfig) {
     let reportName = 'report';
+    if (config.tags.length) {
+      // strip leading @ from tags
+      const tags = sanitizeForFilePath(config.tags.map(tag => tag.substring(1)).join('-')).substring(0, 50);
+      reportName = `${reportName}-${tags}`;
+    }
     if (this._options._commandHash)
```

Comment:
Let's include `tags` into the inputs for the hash - for the case when the tags are long and we may end up with clashing 50 chars prefixes.